examples/stocks/test_driver/scroll_perf.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_driver/driver_extension.dart';
+import 'package:flutter_driver/src/error.dart';
+import 'package:stocks/main.dart' as app;
+
+void main() {
```

Comment:
Yeah, there are other places where I already use `test_driver`. Also, not all the tests are performance tests. Some are simply integration tests. We could use a more generic prefix, perhaps `_e2e.dart`. The issue though, is that `pub run test` has no way to specify a custom prefix. I'll start an email thread to discuss. Should be fixable. We just need to pick something.